packages/blade/src/components/TopNav/TabNav/TabNav.web.tsx

```diff
@@ -16,7 +16,7 @@ import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
 
 const TabNavItems = ({ children, ...props }: BaseBoxProps): React.ReactElement => {
   return (
-    <BaseBox {...props} display="flex" width="100%" gap="spacing.0">
+    <BaseBox {...props} display="flex" width="100%" gap="spacing.0" position="relative" left="-1px">
```

Comment:
Isn't this the component exposed to consumer? we shouldn't spread props and type it as BaseBoxProps no?

Reply:
Wait I'm confused. I see this being used in .stories. But don't see it in our API decision

Reply:
ops nope wait my bad. It is there in API decision. Remove the props spread and BaseBoxProps type I guess.